nightwatch/features/event-reconciliation/unreconciled-events.feature

```diff
@@ -0,0 +1,123 @@
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 1    |
+      | Unexpected outgoing    | 0    |
+
+  Scenario: Unreconciled Check Out Event shows as Unexpected Outgoing and does not affect Availability
+    Given The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+    And I submit the following "check in" event:
+      | centre      | one    |
+      | timestamp   | now    |
+      | cid_id      | 999999 |
+      | person_id   | 12     |
+      | gender      | m      |
+      | nationality | abc    |
+    And I submit the following "check out" event:
+      | centre    | one |
+      | timestamp | now |
+      | person_id | 12  |
+    Then The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 1    |
+      | Unexpected outgoing    | 1    |
+
+  Scenario: (Out Of Order) Unreconciled Check Out Event shows as Unexpected Outgoing and does not affect Availability
+    Given The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+    And I submit the following "check out" event:
+      | centre    | one |
+      | timestamp | now |
+      | person_id | 12  |
+    Given The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
```

Comment:
Not any more

On Wednesday, 4 May 2016, Chris Nesbitt-Smith <notifications@github.com>
wrote:

> In nightwatch/features/event-reconciliation/unreconciled-events.feature
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6#discussion_r62002027>
> :
>
> > +      | Unexpected outgoing    | 0    |
> > +    And I submit the following "check out" event:
> > +      | centre    | one |
> > +      | timestamp | now |
> > +      | person_id | 12  |
> > +    Given The Centre "one" should show the following under "Male":
> > +      | Contractual Capacity   | 1000 |
> > +      | Occupied               | 0    |
> > +      | Beds out of commission | 0    |
> > +      | Contingency            | 0    |
> > +      | Prebookings            | 0    |
> > +      | Availability           | 1000 |
> > +      | Scheduled incoming     | 0    |
> > +      | Scheduled outgoing     | 0    |
> > +      | Unexpected incoming    | 0    |
> > +      | Unexpected outgoing    | 0    |
>
> does it need the check in to get an unexpected outgoing?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6/files/8d568934e372574dcdb57ebb6c0def7d6fba6f57#r62002027>
>